Emby.Server.Implementations/LiveTv/LiveTvDtoService.cs

```diff
@@ -22,6 +22,10 @@ namespace Emby.Server.Implementations.LiveTv
 {
     public class LiveTvDtoService
     {
+        private const string InternalVersionNumber = "4";
+
+        private const string ServiceName = "Emby";
```

Comment:
can we rename here?..

Reply:
I don't think it matters much, it's used for timer IDs.
Maybe in another PR after someone tests if it breaks stuff